tests/python_tests/test_vlm_pipeline.py

```diff
@@ -956,21 +996,38 @@ def test_vlm_pipeline_match_optimum_preresized(request, model_id, image_name, ba
             ],
         }
     ]
+    tokenizer = None
+    if model.config.model_type == "llava-qwen2":
+        processor = get_nanollava_processor()
+        tokenizer = transformers.AutoTokenizer.from_pretrained(model_id, trust_remote_code=True)
 
-    # Gemma3 input_ids has two bos tokens when running with optimum: one in chat template + "add_bos_token" is set to True in tokenizer_config.json
-    if model.config.model_type == "gemma3":
-        processor.tokenizer.add_bos_token = False
+        from optimum.intel.openvino.modeling_visual_language import MODEL_TYPE_TO_CLS_MAPPING
+        preprocess_inputs = MODEL_TYPE_TO_CLS_MAPPING[model.config.model_type].preprocess_inputs
+        inputs = preprocess_inputs(prompt, resized_image, processor, tokenizer, config=model.config)
```

Comment:
Probably we can use `preprocess_inputs` not only for nanoLLaVa but for all other models.
But I believe this is out of scope of this PR

Reply:
Yes, that's what WWB does, so it should be applicable for all models.